sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_eventprocessor_async.py

```diff
@@ -31,7 +31,7 @@
 async def test_loadbalancer_balance():
 
     class MockEventHubClient(object):
-        eventhub_name = "test_eventhub_name"
+        eventhub_name = "test_eventhub_name_{}".format(str(uuid.uuid4()))
```

Comment:
This is a mock event hub client, not really a live test even though it's under the folder livetest. I guess the eventhub name shouldn't have impact on parallel runs because each run will create its own instance and they're different.
We can talk tomorrow.

Reply:
Fair point. I'll just note that the tests hang in this folder 100% of the time in parallel (Windows only). It repros a helluva lot of time on my local box as well.

This latest push should be swapping em to run in serial, so I'm crossing my fingers that it doesn't repro there as well!